### ISSUE TITLE
Feather STM32F405 Express

### ISSUE THREAD

**Issue by Electro-Mike** [adafruit/circuitpython#2864]
Any information on being able to use Circuitpython on the STM32F405 to play wav files?

Thanks.
Mike

**Comment by shalikashvili** [comment]
I would strongly encourage development of the audioio module, as the STM32F405 is one of the more powerful processors in its class, and this module would facilitate applications that would be infeasible on lesser platforms.

**Comment by dastels** [comment]
Agreed, shalikashvili, audio is a pretty common feature for many projects. It's lack on this platform is notable.

**Comment by tannewt** [comment]
We'd love to have it but it isn't a priority for us. We're happy to help anyone get started developing with CircuitPython.

**Comment by iraytrace** [comment]
So, could we encourage Adafuit to update the description at: https://www.adafruit.com/product/4382 to indicate that there is NOT audio support?

Just out of curiosity, what would it take to get this working?  I'd be willing to discuss the possibility of implementing.

**Comment by ladyada** [comment]
the description should not say audio - its been removed.
